src/iuse.cpp

```diff
@@ -951,9 +951,16 @@ int iuse::sleep(player *p, item *it, bool, const tripoint& )
     return it->type->charges_to_use();
 }
 
+int iuse::pblue(player *p, item *it, bool, const tripoint& )
```

Comment:
This function could be implemented using `consume_drug` actor. Same for iodine below. Like this:
```
"use_action" : {
            "type" : "consume_drug",
            "activation_message" : "You take some [drugname].",
            "effects" : [
                {
                    "id": "[drug effect id here]",
                    "duration": [duration here]
                }
            ]
        },
```

This would make maintenance easier.